tests/Predis/Command/CommandTest.php

```diff
@@ -79,7 +79,7 @@ public function testGetArgumentAtIndex(): void
 
         $this->assertEquals($arguments[0], $command->getArgument(0));
         $this->assertEquals($arguments[2], $command->getArgument(2));
-        $this->assertNull($command->getArgument(10));
+        $this->assertEmpty($command->getArgument(10));
```

Comment:
Can you explain this change in return value?

Reply:
This should be changed back to `null` now that I've implemented the changes from `v1.1`, and I just did that.